website/docs/reference/dbt-jinja-functions/zip.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "zip"
+id: "zip"
+---
+
+### zip
+
+The `zip` context method can be used to used to return an iterator of tuples, where the i-th tuple contains the i-th element from each of the argument iterables. ([Python docs](https://docs.python.org/3/library/functions.html#zip))
+        :param 
+        :param 
+        
+__Args__:
+- `*args`: Any number of iterables
+- `default`: A default value to return if `*args` is not iterable
+
+### Usage
+
+```
+{% set my_list_a = [1, 2] %}
+{% set my_list_b = ['alice', 'bob'] %}
+{% set my_zip = zip(my_list_a, my_list_b) | list %}
+{% do log(my_zip) %}  {# [(1, 'alice'), (2, 'bob')] #}
+```
+
+```
+{% set my_list_a = 12 %}
+{% set my_list_b = ['alice', 'bob'] %}
+{% set my_zip = zip(my_list_a, my_list_b, default = []) | list %}
+{% do log(my_zip) %}  {# [] #}
+```
+
+### try_set
```

Comment:
```suggestion
### try_zip
```

Reply:
Totally right. Also will need updating after we rename these to `zip_strict` + `set_strict`: https://github.com/dbt-labs/dbt-core/issues/5475